### PR TITLE
fix: fix design issues from testing

### PR DESCRIPTION
**Pull request by tomzemp**
## Add or between items in shortcut menu [DHIS2-13955](https://dhis2.atlassian.net/browse/DHIS2-13955)

_before_
image

_after_
image

## make bottom bar buttons small [DHIS2-13956](https://dhis2.atlassian.net/browse/DHIS2-13956)

_before_
image

_after_
image

## update styling for Add limits button [DHIS2-13958](https://dhis2.atlassian.net/browse/DHIS2-13958)

_before_
image

_after_
image

## prevent validation tooltip flicker [DHIS2-13954](https://dhis2.atlassian.net/browse/DHIS2-13954)
I think @Birkbjo is including a fix for this in his PR for error handling, but I thought maybe it would be sensible to submit separately as well, so that there's a fix independent of those changes?

_before_
validation_tooltip_before

_after_
validation_tooltip_after

### CODE REVIEW

**Review by cooper-joe — APPROVED** [review]

File: src/data-workspace/data-entry-cell/validation-tooltip.js [hunk 1]
```diff
@@ -25,7 +25,10 @@ const TooltipManager = React.forwardRef(function TooltipManager(
         else {
             close()
         }
-    }, [invalid, active, open, close])
+        // onMouseOut, onMouseOver are not stable references
```
Comment (Mohammer5) [inline src/data-workspace/data-entry-cell/validation-tooltip.js:28]: We should fix that in our UI library. Could you create a Jira ticket?

**Review by Mohammer5 — APPROVED** [review]

File: src/data-workspace/data-entry-cell/validation-tooltip.js (hunk 1 above)
Comment (tomzemp) [reply src/data-workspace/data-entry-cell/validation-tooltip.js:28]: Birk has created one: https://dhis2.atlassian.net/browse/LIBS-359

**Comment by Mohammer5** [comment]
Can be merged IMO

**Review by Birkbjo — APPROVED** [review]

**Comment by Birkbjo** [comment]
> I think @Birkbjo is including a fix for this in his PR for error handling, but I thought maybe it would be sensible to submit separately as well, so that there's a fix independent of those changes?

Fix is identical, so I can just rebase once this is merged.

**Pull request merged**